docker/prod/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG DEV_IMG=dev_vpp_agent
 FROM ${DEV_IMG} as devimg
 
-FROM ubuntu:18.04 as base
+FROM ubuntu:20.04 as base
```

Comment:
VPP 22.02 depends on libmbedtls12 and libmbedcrypto3, which are not available in ubuntu 18.04, so the change to 20.04 is required to build the prod image. (You may notice that I removed the packages libmbedtlsXX and libmbedcryptoX from the dockerfile, because in ubuntu 20.04, they are installed automatically as prerequisites for other packages, so it is no longer necessary to explicitly install them.)

Reply:
ok, so this PR is dependent on https://github.com/ligato/vpp-agent/pull/1839

Reply:
Resolving this because the test check will block merging until https://github.com/ligato/vpp-agent/pull/1839 is resolved.

Reply:
Here is some more info, if you want to go into it.
VPP versions 20.05, 20.09 and 21.06 require libmbedtls10 and libmbedcrypto1 and therefore require ubuntu 18.04.
VPP versions 21.01, 21.10 (unsupported in vpp agent) and 22.02 require libmbedtls12 and libmbedcrypto3 and therefore require ubuntu 20.04.
In this PR I implemented a switch in prod image build script that calls either Dockerfile (ubuntu 20.04) or Dockerfile-legacy (ubuntu 18.04 with libmbedtls10 and libmbedcrypto1), so prod image successfully builds with any supported VPP version.